### ISSUE TITLE
Get Started text discourages participants

### ISSUE THREAD

**Issue by danorys** [mangrove-labs/cutgroup-website#15]
The survey only has 16 questions, many of them selecting an item from a list. 

Yet, the paragraph states _This survey has a lot of questions about where you live, what devices you have, and how you connect to the Internet_.  

I suggest stating _the survey only takes five minutes to complete_ instead.

**Comment by ErnieAtLYD** [comment]
Good point. I did want to say something in regards to the type of questions
folks will be seeing.

Maybe "This survey will only take five minutes to complete and will ask you
questions about where you live, the devices you have and how you connect to
the Internet"?
